public/index.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="Page">
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>React clock</title>
   </head>
-  <body>
+  <body class="Page__body">
```

Comment:
I would still use lowercase in class names)

Reply:
yes